spec/unit/spark_api/models/listing_cart_spec.rb

```diff
@@ -133,4 +133,39 @@
     end
   end
 
+  describe "#listings" do 
```

Comment:
The other tests in this file use this format:
```
context "/listingcarts/<cart_id>", :support do
  on_get_it "should get a listing cart" do
    ...
  end
end
```
The way I've done it makes more sense to me, but if some can explain why it should be done like this example, I'd be happy to change it.